Dockerfile

```
@@ -14,11 +14,16 @@ RUN dnf install -y tar gcc make && \
     dnf clean metadata packages && \
     arch=$(arch | sed s/aarch64/arm64/ | sed s/x86_64/amd64/) && \
     curl -Ls https://go.dev/dl/${GO_VER}.linux-${arch}.tar.gz |tar -C /usr/local -xzf -  && \
-    useradd -U -u 1000 -m -d /home/${CONTAINERUSER} -s /usr/bin/bash ${CONTAINERUSER}
+    useradd -U -u 1000 -m -d /home/${CONTAINERUSER} -s /usr/bin/bash ${CONTAINERUSER} && \
+    mkdir /eco-gotests && chown ${CONTAINERUSER}:${CONTAINERUSER} /eco-gotests
 
 USER ${CONTAINERUSER}
 WORKDIR /home/${CONTAINERUSER}
 RUN go install github.com/onsi/ginkgo/v2/${GINKGO_VER}
+ENV GOPATH="/home/${CONTAINERUSER}/go"
+ENV PATH="$PATH:$GOPATH/bin"
+
+WORKDIR /eco-gotests
```

Instead of /eco-gotests can we put the path under the test user directory?
```suggestion
WORKDIR /home/${CONTAINERUSER}/eco-gotests
```

---

This will impact the OVRD (in ocp-eco-gotests.groovy) - ${CONTAINERUSER} is unknown there
/ is better.

---

We maintain both so it's safe to assume the user. If it isn't dynamic enough you can use this:
```
$ skopeo inspect docker://quay.io/ocp-edge-qe/eco-gotests:latest | jq -r '.Labels["container.user"]'
testuser
```

---

do u mean that inside container workdir would be $HOME/eco-gotests ?

---

podman inspect $ECO_GOTESTS_IMAGE | jq -r '.[].Config.WorkingDir'